lte/cloud/go/services/lte/obsidian/handlers/handlers_test.go

```diff
@@ -3350,7 +3351,7 @@ func TestAPNResource(t *testing.T) {
 		ParamNames:             []string{"network_id"},
 		ParamValues:            []string{"n0"},
 		ExpectedStatus:         500, // this would actually make more sense as a 400, but it's a non-trivial fix
-		ExpectedErrorSubstring: `could not find entities matching [type:"apn" key:"apn0" ]`,
+		ExpectedErrorSubstring: "could not find entities matching" + tests.Separator + "[type:\"apn\"" + tests.Separator + "key:\"apn0\"]",
```

Comment:
is this change triggered by the linter?

Reply:
The problem here was that the tests are failing because an extra whitespace between "type" and "key" fields for example. This is not fixed by removing the whitespace since it appears somewhat randomly. We think it is similar to this issue https://github.com/golang/protobuf/issues/1269. The inserted separator is the workaround.

The comparison now takes the expected error (sub)string, splits it at the separator, and checks wether the parts are contained in the actual message.